designs/developer/operator-pipelines-47/yaml/index.md

```diff
@@ -0,0 +1,13 @@
+---
+parent: Developer
+version: 4.7
+---
+
+# Edit YAML in Pipeline Builder
+
+## Description
+Being consistent with other areas of the console that use radio buttons to switch between form fields and YAML, the user can switch between the Pipeline Builder and the YAML view using radio buttons. Unlike other areas in the console, the radio button label will specifically say Pipeline Builder. The user can freely switch between the two views without losing changes between them. Any data that was entered in the form would be reflected in the YAML view, as well as any data entered in the YAML would be reflected in the form if it is switched back to (as its fields support).
+![form](img/image3.png)
```

Comment:
Were we able to update the tech preview badge to use the latest convention or is that still a hope for a future release (4.8 maybe)? I think we wanted it to look less like a button and more like the badge/label conventions but unsure the timing. cc @beaumorley @bmignano @serenamarie125

Reply:
@beanh66 We talked about updating the badges for 4.7. But we need Docs to be able to update any screens they have captured. I believe this is still the outstanding. cc @gdoyle1